### PR TITLE
[WorksForYou] UI Fixes

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] Disabled iPhone autorotation
- [x] Changed some padding
- [x] Added tap gesture to # of works label (to push artist VC)
- [x] More UI tweaks

Related to #1230

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>PR is classed as Work in Progress</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARArtworkFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkFavoritesNetworkModelSpec</a> in <code>test_getFavorites__makes_request_if_no_request_is_in_progress</code></li>
<li><a href="https://github.com/search?q=ARArtworkViewControllerBuyButton+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkViewControllerBuyButtonSpec</a> in <code>test_buy_button__posts_order_if_artwork_has_no_edition_sets</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__passes_artwork_to_share_controller</code></li>
<li><a href="https://github.com/search?q=ARFairViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARTopMenuViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARTopMenuViewControllerSpec</a> in <code>test_navigation__favorites__logged_in__tab_behavior__already_selected__pops_to_root</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

File: Artsy/View_Controllers/WorksForYou/ARWorksForYouNotificationItemViewController.m [hunk 1]
```diff
@@ -112,5 +114,15 @@ - (void)didSelectArtist:(Artist *)artist animated:(BOOL)animated
     [self.navigationController pushViewController:artistVC animated:animated];
 }
 
+#pragma mark - ARArtworkMasonryLayoutProvider
+- (ARArtworkMasonryLayout)masonryLayoutForSize:(CGSize)size
+{
+    if (self.artworksVC.items.count > 1) {
+        return [UIDevice isPad] && (size.width > size.height) && self.artworksVC.items.count >= 3 ? ARArtworkMasonryLayout3Column : ARArtworkMasonryLayout2Column;
```
Comment (mennenia) [inline Artsy/View_Controllers/WorksForYou/ARWorksForYouNotificationItemViewController.m]: Did we move away from size classes? I thought that was going to be how we determined devices/sizes from now on :o

**Comment by mennenia** [comment]
Beautiful spacing on the snapshots! <3

File: Artsy/View_Controllers/WorksForYou/ARWorksForYouNotificationItemViewController.m (hunk 1 above)
Comment (alloy) [inline Artsy/View_Controllers/WorksForYou/ARWorksForYouNotificationItemViewController.m]: Should use size classes indeed.

**Comment by orta** [comment]
f289d1af878

**Comment by mennenia** [comment]
new stuff is looking great. Wanna change to size classes and then I'll merge? :)

File: Artsy/View_Controllers/WorksForYou/ARWorksForYouNotificationItemViewController.m [hunk 2]
```diff
@@ -112,5 +114,15 @@ - (void)didSelectArtist:(Artist *)artist animated:(BOOL)animated
     [self.navigationController pushViewController:artistVC animated:animated];
 }
 
+#pragma mark - ARArtworkMasonryLayoutProvider
+- (ARArtworkMasonryLayout)masonryLayoutForSize:(CGSize)size
+{
+    if (self.artworksVC.items.count > 1) {
+        return self.traitCollection.horizontalSizeClass == UIUserInterfaceSizeClassRegular && self.artworksVC.items.count >= 3 ? ARArtworkMasonryLayout3Column : ARArtworkMasonryLayout2Column;
```
Comment (sarahscott) [inline Artsy/View_Controllers/WorksForYou/ARWorksForYouNotificationItemViewController.m]: @mennenia did you mean this line? Or are there any other places where I didn't use size classes and should have?

File: Artsy/View_Controllers/WorksForYou/ARWorksForYouViewController.m [hunk 3]
```diff
@@ -144,4 +194,9 @@ - (void)removeLoadingIndicator
     }
 }
 
+- (BOOL)shouldAutorotate
+{
+    return [UIDevice isPad];
```
Comment (orta) [inline Artsy/View_Controllers/WorksForYou/ARWorksForYouViewController.m]: I imagine Maxime is thinking this, you can use these functions: https://github.com/artsy/eigen/commit/a607a951d763f8831e101d768270fd78305296d8#diff-cec4aab5e08b6c0c59a4f60a30b88ab1R3

Comment (sarahscott) [inline Artsy/View_Controllers/WorksForYou/ARWorksForYouViewController.m]: :+1:

**Comment by orta** [comment]
Build fail is danger, and known.

**Comment by mennenia** [comment]
<3

**Pull request merged**